.github/workflows/pr.yml

```
@@ -0,0 +1,31 @@
+name: Windows CI
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  run-windows-tests:
+    name: Build & run tests
+    runs-on: windows-2022
+    strategy:
+      matrix:
+        configuration: ['debug', 'release']
+        platform: ['x86', 'x64']
+
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout Code
```

```suggestion
      - name: Checkout Code
        uses: actions/checkout@v2
```
Just for consistency with the others